install-storageos/020-install-storageos.md

```diff
@@ -0,0 +1,100 @@
+
+First we need to install the StorageOS Operator using the following yaml manifest.
+
+`kubectl create -f https://github.com/storageos/cluster-operator/releases/download/v2.0.0/storageos-operator.yaml`{{execute}}
+
+Verify the Cluster Operator pod status
+
+`kubectl -n storageos-operator get pod -w`{{execute}}
+
+> The READY 1/1 indicates that `storageoscluster` resources can be created. Press `Ctrl+C` to continue once the pod is up.
+
+Create a Secret defining the StorageOS credentials.
```

Comment:
Is it worth explaining that the credentials are base64 encoded and are really just `storageos`?